docs/traffic-policy/actions/redirect/examples/1-basic-example.mdx

```diff
@@ -0,0 +1,47 @@
+import ConfigExample from "/src/components/ConfigExample.tsx";
+
+### Redirect using Paths
+
+The following [Traffic Policy](/docs/http/traffic-policy/index.mdx) 
+configuration demonstrates how to use the `redirect` action to redirect incoming 
+requests from `/products` to `/store/products`.
+
+#### Example Traffic Policy Document
+
+<ConfigExample
+  snippetText={null}
+  showLineNumbers={true}
+  jsonMetastring="{5-12}"
+  yamlMetastring="{4-13}"
+  config={{
+    inbound: [{
+      actions: [
+				{
+					type: "redirect",
+					config: {
+						from: "/products",
+						to: "/store/products",
+					},
+				},
+      ],
+    }]
+  }}
+/>
+
+This configuration will redirect any request from `/products` to `/store/products` 
+with the default `302 Found` status code.
+
+#### Example Request
+
+```shell
+$ curl -i https://example.ngrok.app/products
+```
+
+```http
+HTTP/1.1 302 Found
+Location: https://example.ngrok.app/store/products
```

Comment:
`Location` -> `location` (lowercase key), i think also relevant for other examples here

Reply:
Locally, this was uppercase, but I can lowercase it